renderer/components/Onboarding/Steps/WalletCreate.js

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { FormattedMessage } from 'react-intl'
 import { Spinner, Text } from 'components/UI'
+import CenteredContent from 'components/UI/CenteredContent'
```

Comment:
This should probably be combined into the above named imports from `components/UI`